stdlib/2/os/__init__.pyi

```diff
@@ -144,66 +145,66 @@ def fsync(fd: int) -> None: ...
-def lchflags(path: unicode, flags: int) -> None: ...
-def lchmod(path: unicode, uid: int, gid: int) -> None: ...
-def lchown(path: unicode, uid: int, gid: int) -> None: ...
-def link(source: unicode, link_name: unicode) -> None: ...
+def chflags(path: _PathType, flags: int) -> None: ...
+def chroot(path: _PathType) -> None: ...
+def chmod(path: _PathType, mode: int) -> None: ...
+def chown(path: _PathType, uid: int, gid: int) -> None: ...
+def lchflags(path: _PathType, flags: int) -> None: ...
+def lchmod(path: _PathType, uid: int, gid: int) -> None: ...
+def lchown(path: _PathType, uid: int, gid: int) -> None: ...
+def link(source: _PathType, link_name: _PathType) -> None: ...
 def listdir(path: AnyStr) -> List[AnyStr]: ...
-def lstat(path: unicode) -> Any: ...
-def mkfifo(path: unicode, mode: int = ...) -> None: ...
-def mknod(filename: unicode, mode: int = ..., device: int = ...) -> None: ...
+def lstat(path: _PathType) -> Any: ...
+def mkfifo(path: _PathType, mode: int = ...) -> None: ...
+def mknod(filename: _PathType, mode: int = ..., device: int = ...) -> None: ...
 def major(device: int) -> int: ...
 def minor(device: int) -> int: ...
 def makedev(major: int, minor: int) -> int: ...
-def mkdir(path: unicode, mode: int = ...) -> None: ...
-def makedirs(path: unicode, mode: int = ...) -> None: ...
-def pathconf(path: unicode, name: str) -> str: ...
+def mkdir(path: _PathType, mode: int = ...) -> None: ...
+def makedirs(path: _PathType, mode: int = ...) -> None: ...
+def pathconf(path: _PathType, name: str) -> str: ...
 def readlink(path: AnyStr) -> AnyStr: ...
-def remove(path: unicode) -> None: ...
-def removedirs(path: unicode) -> None: ...
-def rename(src: unicode, dst: unicode) -> None: ...
-def renames(old: unicode, new: unicode) -> None: ...
-def rmdir(path: unicode) -> None: ...
-def stat(path: unicode) -> Any: ...
+def remove(path: _PathType) -> None: ...
+def removedirs(path: _PathType) -> None: ...
+def rename(src: _PathType, dst: _PathType) -> None: ...
+def renames(old: _PathType, new: _PathType) -> None: ...
+def rmdir(path: _PathType) -> None: ...
+def stat(path: _PathType) -> Any: ...
 @overload
 def stat_float_times(newvalue: bool = ...) -> None: ...
 @overload
 def stat_float_times() -> bool: ...
-def statvfs(path: unicode) -> _StatVFS: ...
-def symlink(source: unicode, link_name: unicode) -> None: ...
-def unlink(path: unicode) -> None: ...
-def utime(path: unicode, times: Optional[Tuple[int, int]]) -> None: ...
-def walk(top: AnyStr, topdown: bool = ..., onerror: Any = ...,
-         followlinks: bool = ...) -> Iterator[Tuple[AnyStr, List[AnyStr],
-                                                    List[AnyStr]]]: ...
+def statvfs(path: _PathType) -> _StatVFS: ...
+def symlink(source: _PathType, link_name: _PathType) -> None: ...
+def unlink(path: _PathType) -> None: ...
+def utime(path: _PathType, times: Optional[Tuple[int, int]]) -> None: ...
+def walk(top: _PathType, topdown: bool = ..., onerror: Any = ...,
```

Comment:
AnyStr is actually correct here: os.walk returns bytes if you give it bytes and Text if you give it Text. Besides, we should generally avoid Unions in return types.